about.html

```diff
@@ -0,0 +1,139 @@
+        <p>The most important Tango event in Southamerica. The Internatiuonal Tango Festival will be held in Bogota, Colombia. The most famous milongueros from Colombia will present in this magnificent event.</p>
+      </div>
+    </section>
+
+    <main>
+      <section id="about-body">
+        <div class="contact-box">
+          <p class="contact-about">Please contact us per email for any further questions about International Tango Festival 2022! </p>
+          <p class="email">intertangofest2022@email.com</p>
+        </div>
+        <h2 class="program-heading">International Tango Festival 2022 Logo</h2>
+        <div class="line"></div>
+
+        <div class="white-box scale-2">
+          <img src="./src/img/project-logo/logo_tango_194x54.png" alt="tango festival logo">
+        </div>
+        
+        <div class="section-divider mobile"></div>
+        
+        <div class="past-conferences" id="past">
+          <h2 class="program-heading">See the past International Tango Festival</h2>
+          <div class="line"></div>
+          <p>Take a look to the last two International Tango Festivals which took place in Buenos Aires and Montevideo </p>
+          <div class="conferences-container">
+            
+            <div class="container">
+              <div class="text-container">
+                <h3 class="img-text img-header">2021</h3>
+                <p class="img-text img-place">International Tango Festival in Mexico City</p>
+              </div>
+              <div class="bg">
+                <img src="./src/img/conferences/conference-1.jpg" alt="tango festival image" width="350" height="250">
+              </div>
+
+            </div>
+
+            <div class="container">
+              <div class="text-container">
+                <h3 class="img-text img-header">2020</h3>
+                <p class="img-text img-place">International Tango Festival in Montevideo</p>
+              </div>
+              <div class="bg">
+                <img src="./src/img/conferences/conference-2.jpg" alt="tango festival image" width="350" height="250">
+              </div>
+
+            </div>
+
+          </div>
+            
+          </div>
+        </div>
+      </section>
+      <!-- sponsors section -->
+      <section class="about-sponsors" id="sponsors">
+        <h3 class="sponsors-heading">Sponsors</h3>
+        <div class="line"></div>
+        <div class="sponsors-logos">
+          <img class="google-logo" src="./src/img/sponsors/sponsor-1.png" alt="Google logo" width="200" height="200">
+          <img src="./src/img/sponsors/sponsor-2.png" alt="Airbnb logo" width="200" height="100">
+          <img src="./src/img/sponsors/sponsor-3.png" alt="GitHub logo" width="100" height="100">
+          <img src="./src/img/sponsors/sponsor-4.png" alt="Firefox logo" width="100" height="100">
+          <img src="./src/img/sponsors/sponsor-5.png" alt="Micorvese Logo" width="400" height="100">
+        </div>
+      </section>
```

Comment:
- For the about page, this section isn't included in the design template. Kindly remove this section 🙏 🙏. Remember it is important that you try to replicate the Figma design as closely as possible to give confidence to recruiters in your ability to produce exact business requirements 🙏

Design Template:
<img width="554" alt="about behance footer" src="https://user-images.githubusercontent.com/73607512/197244798-b5208603-dab0-4cfd-a24f-21b15ab6fb2e.PNG">

Reply:
ok